azure-pipelines.release-fluentui.yml

```diff
@@ -1,74 +1,220 @@
 pr: none
 trigger: none
 
-# Customize build number to include major version
-# Example: fluentui_20190626.1
-name: 'fluentui_$(Date:yyyyMMdd)$(Rev:.r)'
+schedules:
+  - cron: 0 3 * * *
+    branches:
+      include:
+        - master
+
```

Comment:
At the top level please add:
```yml
workspace:
  clean: all
```

And this at the end of each job:
```yml
  - task: DeleteFiles@1
    inputs:
      SourceFolder: $(Build.SourcesDirectory)
      Contents: '**/*'
    condition: always()
```

Both of these are super important for reducing "Directory not empty" build failures.

Reply:
Hmm actually I wonder if the issue is that there's something else we need to be cleaning in all our jobs *besides* `Build.SourcesDirectory`... That's for a different PR though.

Reply:
Added 👍

Reply:
I found out `workspace` is not valid for multi-job pipeline, so I added one for each job